core/src/views/textbox.rs

```diff
@@ -344,12 +344,13 @@ impl Model for TextboxData {
             }
 
             TextEvent::Submit => {
-                cx.emit(TextEvent::EndEdit);
+                println!("{}", self.text);
```

Comment:
This should probably be removed or is there a good reason for it to be there?

Reply:
Yep my bad, needed to debug something.